lldb/source/Plugins/TypeSystem/Swift/TypeSystemSwiftTypeRef.cpp

```diff
@@ -2490,6 +2490,14 @@ CompilerType TypeSystemSwiftTypeRef::GetChildCompilerTypeAtIndex(
     bool &child_is_base_class, bool &child_is_deref_of_parent,
     ValueObject *valobj, uint64_t &language_flags) {
   LLDB_SCOPED_TIMER();
+  child_name = "";
+  child_byte_size = 0;
+  child_byte_offset = 0;
+  child_bitfield_bit_size = 0;
+  child_bitfield_bit_offset = 0;
+  child_is_base_class = false;
+  child_is_deref_of_parent = false;
+  language_flags = 0;
```

Comment:
Why are these reset here? If this is needed, it's missing a `#ifndef NDEBUG`

Reply:
It's because we'll validate them against their ast_* counterparts.
They are initialized unconditionally because the initialization has an effect outside of the function (these are passed in  by reference), I really don't want to change LLDB's behavior in asserts/non-asserts build.